### ISSUE TITLE
Illegal Position in the Penalty Area

### ISSUE THREAD

**Issue by ahasselbring** [RoboCup-SPL/Rules#65]
The current Illegal Position rule dates back to times when only the goalkeeper was allowed to be in its team's penalty area (coinciding with today's goal area). I assume that the main motivation for this is to prevent teams from, either intentionally or unintentionally, blocking the goal and making it physically impossible for the opponent to score, which would not be possible in real soccer (and therefore, such a rule does not exist there).
In 2014, one field player was allowed to enter its team's penalty area (in addition to the goalkeeper), and in 2015, the rule was changed to any two players of the defending team, regardless of the status as field player or goalkeeper (although section 2.3 wasn't adapted until 2018).
In 2018, the illegal defender rule was extended to all game states (i.e. including ready and set, while previously only being applied during the playing state).
Then, for the planned 2020 competition, the penalty area was enlarged and consequently, the 2 defender limit was increased to 3, while also limiting the number of attackers in a penalty area to 3 (as it was assumed that otherwise the attackers can too easily outnumber the defenders in the penalty area). The later introduction of the goal area (with the dimensions of the old penalty area) didn't revert this rule change.

Given that such a rule does not exist in real soccer, I think it has pretty much diverged from its original purpose. Applying the rule in the ready state does not make sense to me. At the moment, a lot of Illegal Position penalties are given in situations where there is no danger that the additional player prevents progress in the game (e.g. two defenders and the goalkeeper are in the penalty area oriented towards one side where the ball is, and from the other side, a player that has just returned from a penalty crosses the corner of the penalty area - this is illegal but does not impact the relevant situation at the other side of the penalty area at all). You could argue that they *would* interfere later if they weren't penalized, but I think this takes away too much freedom (that real soccer players have) for a questionable benefit. You could also argue that the current rule reduces the cognitive load on referees since they can avoid congested situations before they occur, but that could apply anywhere on the field and not just in the penalty areas.

In my experience, in situations in which players inadvertently block the ball/goal completely, this may cause some delay, but sooner or later some of them are going to be removed anyway for other illegal behavior (e.g. falling and failing to get up due to other close robots, pushing, walking into a goal post). The only situation left would be if a team (e.g. after being ahead by one goal) deliberately builds a "wall" of robots in front of the goal. I still don't think this is a successful strategy, but it is simply an example of unsportsmanlike behavior.

In summary, I propose the following three options (apart from not changing the rules), with (3) being my personal favorite:
1. Remove illegal position restrictions in the ready state (and penalize robots only when they are still illegal in the set state)
2. (1) + revert to the old (pre-2020) meaning (max. 2 defenders in goal area, no restrictions for attackers)
3. Remove illegal position restrictions in the penalty area entirely (except during penalty kicks)

**Comment by PatrickGoettsch** [comment]
You are right, it would make it a lot easier for the referees. (3) Sounds good, except for the point of unsportsmanlike behavior. How to handle this?

**Comment by ahasselbring** [comment]
I have left a limit for 3 defenders in the goal area during set/playing (since I believe that was our conclusion on January 6).

Another option would be to remove first, and reserve the right to reintroduce it after seeing what happens at a local competition (if there is one).

**Comment by ahasselbring** [comment]
I think the current state is okay.
